scripts/functional_tests.js

```diff
@@ -1,2 +1,5 @@
 require('../src/babel-register');
-require('../packages/kbn-test').runTestsCli();
+require('../packages/kbn-test').runTestsCli([
+  'test/functional/config.js',
```

Comment:
IMO, these (and the scripts below) should be using absolute paths, otherwise the default configs won't resolve properly if you call this script with a CWD other than the repository root. The easiest method for this and other scripts is probably replacing relative path with `require.resolve('../tests/functional/config')`.